libs-scala/resources-akka/src/main/scala/com/digitalasset/resources/akka/SourceQueueResourceOwner.scala

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.resources.akka
+
+import akka.stream.Materializer
+import akka.stream.scaladsl.{RunnableGraph, SourceQueue, SourceQueueWithComplete}
+import com.daml.resources.{AbstractResourceOwner, HasExecutionContext, ReleasableResource, Resource}
+
+import scala.concurrent.Future
+
+class SourceQueueResourceOwner[T, Context: HasExecutionContext](
+    queueGraph: RunnableGraph[SourceQueueWithComplete[T]]
+)(implicit
+    materializer: Materializer
+) extends AbstractResourceOwner[Context, SourceQueue[T]] {
+  override def acquire()(implicit context: Context): Resource[Context, SourceQueue[T]] =
+    ReleasableResource(Future(queueGraph.run()))(queue =>
```

Comment:
`Resource.apply` delegates to `ReleasableResource`.

```suggestion
    Resource(Future(queueGraph.run()))(queue =>
```

Reply:
I think the resource you're referring to comes from `ledger/ledger-resources`.  The owners from this package use the `ReleasableResource`, are you fine to keep it, @SamirTalwar-DA?

Reply:
Yeah, sure, if it's easier.